test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -25,13 +25,53 @@ Class. We use :proc:`~UnitTest.runTest` to pass the tests.
 .. code-block:: chapel
 
    use UnitTest;
+    
+   // calculates factorial
+   proc factorial(x: int): int {
+     return if x == 0 then 1 else x * factorial(x-1); 
+   }
 
    proc test1(test: Test) throws {
-     test.assertTrue(True);
+     test.skipIf(factorial(0) != 1,"Base condtion is wrong in factorial");
```

Comment:
Could we add another test to this example that demonstrates the unconditional `skip` as well?

Reply:
Added a new example